clang/test/Sema/complex-arithmetic.c

```diff
@@ -21,6 +25,7 @@ void a() {
   EVALF((2. + 3i) + (4. + 5i), 6. + 8i);
   EVALF((2. + 3i) - (4. + 5i), -2. - 2i);
   EVALF((2. + 3i) * (4. + 5i), -7. + 22i);
+  // div-precision-warning@+1 {{excess precision is requested but the target does not support excess precision which may result in observable differences in arithmetic behavior ,additional use of 'long double' were found but not diagnosed}}
```

Comment:
This shows there's a formatting issue in the diagnostic because of:
```
behavior ,additional use
```

Also, I don't see any `long double` variables involved, these are all `double`. Are we passing in the wrong type or am I just blind?

Reply:
We may want to edit the diagnostic and edit this:
`additional use of`
to
`additional use of requested higher precision`
WDYT?

Reply:
Yeah, something like `additional uses where the requested higher precision cannot be honored were found but not diagnosed` maybe? Then we drop the type information entirely, which was what was causing my confusion.